src/helpers/service-account/service-account-helper.ts

```diff
@@ -28,7 +28,7 @@ export interface ServiceAccount {
 }
 
 export interface ServiceAccountsPayload {
-  data: ServiceAccount[];
+  data: (ServiceAccount & { clientId: string })[];
```

Comment:
This is kind of confusing to have clientID and clientId in the same payload, everything else looks good

Reply:
It is, but the RBAC API currently retrieves us ServiceAccounts with `clientID`  and https://sso.stage.redhat.com/auth/realms/redhat-external/apis/service_accounts gives us `clientId`. I think it is better than introducing another interface for the same thing. I'll talk to backend folks to get rid of it